### PR TITLE
feat: custom config and font for funke

### PR DESCRIPTION
**Pull request by TimoGlastra**
Builds on top of #120. Adds custom config for the Funke wallet with custom primary color and fonts. The font setup has been improved in general by loading them in the app config, which pre-loads them and means we don't have to load the fonts on startup.

Paradym wallet still uses Inter, Funke wallet uses Raleway.

@janrtvld Maybe you can help a bit with configuring the theme and colors / sizes for the Funke app. I only configured the primary color now.

### CODE REVIEW

**Comment by TimoGlastra** [comment]
This branch also contains some fixes for #120 . so we either need to merge at the same time, or merge this PR into #120 first

File: apps/funke/app.config.js [hunk 1]
```diff
@@ -52,7 +52,21 @@ const config = {
...
-  plugins: ['expo-font', 'expo-secure-store', 'expo-router'],
+  plugins: [
+    [
+      'expo-font',
+      {
+        fonts: [
+          '../../node_modules/@expo-google-fonts/raleway/Raleway_400Regular.ttf',
+          '../../node_modules/@expo-google-fonts/raleway/Raleway_500Medium.ttf',
+          '../../node_modules/@expo-google-fonts/raleway/Raleway_600SemiBold.ttf',
+          '../../node_modules/@expo-google-fonts/raleway/Raleway_700Bold.ttf',
```
Comment (janrtvld) [inline apps/funke/app.config.js:63]: We actually use 'Raleaway' for only the titles and headings. The body font is 'Open sans'.

File: apps/funke/tamagui.config.ts [hunk 2]
```diff
@@ -1,13 +1,46 @@
...
+      ...tokens.color,
+    },
+  },
+})
+
+type ConfIg = typeof config
 declare module 'tamagui' {
   // eslint-disable-next-line @typescript-eslint/no-empty-interface
-  interface TamaguiCustomConfig extends Conf {}
+  interface TamaguiCustomConfig extends ConfIg {}
```
Comment (janrtvld) [inline apps/funke/tamagui.config.ts:43]: Why this change?

File: apps/funke/app.config.js (hunk 1 above)
Comment (TimoGlastra) [reply apps/funke/app.config.js:63]: Okay will update 👍

File: apps/funke/tamagui.config.ts (hunk 2 above)
Comment (TimoGlastra) [reply apps/funke/tamagui.config.ts:43]: will revert

Comment (TimoGlastra) [reply apps/funke/tamagui.config.ts:43]: Reverted in another PR

File: apps/funke/app.config.js (hunk 1 above)
Comment (TimoGlastra) [reply apps/funke/app.config.js:63]: I will do this in a separate PR, easier and it's not an issue for the Funke wallet
